### ISSUE TITLE
Listings Management Fieldset: Application Dates

### ISSUE THREAD

**Issue by slowbot** [bloom-housing/bloom#1182]
**Goal**: Leasing agent should be able to add events to a listing

**Fields**:
Field Name | Entity Model | Type | Current State | Required
------------ | ------------- | --- | ----- | -----
Application Start Date | applicationStartDate | date | Yes | Yes
Application Start Time | | date | No | Yes
Application Due Date | applicationDueDate | date | Yes | Yes
Application Due Time | | date | No | Yes
Open House Date | | date | No | Yes
Open House Start Time | | date | If selected | Yes
Open House End Time | | date | If selected | Yes
URL | | date | ? | No

**Acceptance Criteria**
- If user adds Open House, open modal with Open House date, times and URL fields
- Once saved, display summary of event in Open House table

**Questions**
- Can we autofill the time fields? Start Date 9am an Due Date 5pm

**Figma**:
Default:
https://www.figma.com/file/Y8bFd0aSz7eYJ2TCNid6s3/bloom-partners?node-id=2726%3A127598

Filled:
https://www.figma.com/file/Y8bFd0aSz7eYJ2TCNid6s3/bloom-partners?node-id=2726%3A127818

Modal:
https://www.figma.com/file/Y8bFd0aSz7eYJ2TCNid6s3/bloom-partners?node-id=2736%3A126754

Open House:
https://www.figma.com/file/Y8bFd0aSz7eYJ2TCNid6s3/bloom-partners?node-id=2736%3A126865

**Comment by pbn4** [comment]
Field Name | Entity Model | Type | Current State | Required | Backend review comment | 
------------ | ------------- | --- | ----- | ----- | --- |
Application Start Date | applicationStartDate | date | Yes | Yes | Exists as `applicationOpenDate` and containts both date and time |
Application Start Time | | date | No | Yes | As described above |
Application Due Date | applicationDueDate | date | Yes | Yes | Contains  both date and time  | 
Application Due Time | | date | No | Yes | As described above |
Open House Date | | date | No | Yes | Defined in `listing.events[]` array. See ListingEvent definition. \*   |
Open House Start Time | | date | If selected | Yes | As described above  |
Open House End Time | | date | If selected | Yes |  As described above |
URL | | date | ? | No |  As described above |

ListingEvent properties: 

```
type: ListingEventType 
startTime?: Date
endTime?: Date
url?: string | null
note?: string | null
label?: string | null
```

ListingEventType: 

```
export enum ListingEventType {
  openHouse = "openHouse",
  publicLottery = "publicLottery",
  lotteryResults = "lotteryResults",
}
```

**Comment by pbn4** [comment]
>Can we autofill the time fields? Start Date 9am an Due Date 5pm
I guess we could do this frontend side with the current schema (cc: @dominikx96 )

**Comment by dominikx96** [comment]
@pbn4 @slowbot yes, I think it could be possible with this schema (if both: date and time fields are required)

**Comment by seanmalbert** [comment]
@pbn4, @dominikx96  and @slowbot ,

I think it makes sense to have timezone set at the jurisdiction level. So, if we're going to have a jurisdiction table (what we've mostly been referring to as county so far), timezone can be a field in the table. It seems like we don't need to worry about that until we get jurisdictions in different timezones, but we should at least be pulling the timezone value from the backend and not have it hardcoded anywhere in the frontend apps.

**Comment by kathyccheng** [comment]
Closed in #1382
